frameworks/cassandra/universe/config.json

```diff
@@ -253,6 +253,58 @@
                   "minimum": 128000
                 }
               }
+            },
+            "rlimit_stack": {
+              "description": "Specifies RLIMIT_STACK, a value one greater than the maximum size of the initial thread's stack, in bytes.",
+              "type": "object",
+              "properties": {
+                "soft" : {
+                  "type": "integer",
+                  "description": "The  soft  limit  is  the  value that the kernel enforces for the corresponding resource.",
```

Comment:
Spacing seems off here. Looks like there are two spaces in between `The soft limit is the value`. Please fix.

Reply:
Fixed